client/Components/GetInTouchForm.js

```diff
@@ -0,0 +1,107 @@
+import React, { useState } from 'react';
+
+const GetInTouchForm = ( { handleFormChange }) => {
+
+  const [isHidden, toggleHidden] = useState(true);
```

Comment:
would like to figure out way to toggle hidden on/off for individual inputs without having multiple `useState` functions

Reply:
You can create an object with useState that has a prop for each input status